### PR TITLE
Profile screen

### PR DESCRIPTION
**Pull request by MuresanCoralia**
### Summary

Profile screen (Xib)

### Metadata
- Ticket link: [Profile Screen](https://trello.com/c/6sSszAJJ/12-ui-profile-screen)
- Relevant design:

### Checks
- [ ] Team members are added to review the Pull Request
- [ ] Dead/unused code is deleted?
- [ ] PR does not break existing functionality
- [ ] Implementation matches the description in the story
- [ ] Images/Gifs have been added for all UI related changes

### CODE REVIEW

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/PProfileViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
+//
+//  PProfileViewController.swift
+//  Internship2022TeamPatri
+//
+//  Created by Coralia Diana Muresan on 08.08.2022.
+//
+
+import UIKit
+
+class PProfileViewController: UIViewController {
```
Comment (carmenj) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/PProfileViewController.swift]: typo - "PProfileViewController". Same for the corresponding xib

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Login/LogInViewController.swift [hunk 2]
```diff
@@ -119,6 +119,8 @@ class LogInViewController: UIViewController {
         self.navigationController?.navigationBar.tintColor = UIColor.colorText
         self.navigationController?.navigationBar.backIndicatorTransitionMaskImage = backButtonArrow
         self.navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: UIBarButtonItem.Style.plain, target: nil, action: nil)
+        //let profileViewController = PProfileViewController.loadFromNib()
+        //self.navigationController?.pushViewController(profileViewController, animated: true)
```
Comment (carmenj) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Login/LogInViewController.swift]: are these still needed?

File: Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/PProfileViewController.swift [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        // adding the title
+        let lbNavTitle = UILabel (frame: CGRect(x: 0, y: 0, width: 320, height: 50))
+            lbNavTitle.textColor = UIColor.colorText
+            lbNavTitle.numberOfLines = 0
+            lbNavTitle.center = CGPoint(x: 0, y: 0)
+            lbNavTitle.textAlignment = .center
+            lbNavTitle.text = title
+            lbNavTitle.font = lbNavTitle.font.withSize(32)
+        self.navigationItem.titleView = lbNavTitle
+    }
```
Comment (carmenj) [inline Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/PProfileViewController.swift]: The same code is also present in `CoursesViewController`. Check the PR created by Eduard: https://github.com/Practica-iOS-2021/Internship2022TeamPatri/pull/14/files
Maybe you can find a solution together with Eduard to reuse the code (without duplicating it).

Comment (MuresanCoralia) [reply Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/Profile/PProfileViewController.swift]: I made the code and tried to make an extension and I couldn't. So I talked to him to use the same function so the screens look the same until we make the connections and put it maybe in a tab bar controller

**Pull request closed without merging**